README.md

```diff
@@ -1,47 +1,119 @@
-# Cell morphology analysis
+# Phenotypic Analysis of Interfertile Algal Species
 
 This repository contains code for image processing and analysis to compare phenotypes of interfertile *Chlamydomonas* species. These analyses are included in the publication [Phenotypic differences between interfertile Chlamydomonas species](https://doi.org/10.57844/arcadia-35f0-3e16).
 
 ![Chlamydomonas smithii is larger than C. reinhardtii but they have a similar shape](https://github.com/Arcadia-Science/chlamy-comparison/assets/110641190/a6854e63-e4f6-4779-904e-02ca65fb4d4d)
 
-Release v1.0.1
+Release v2.1
+
+Github
 
 [![DOI](https://zenodo.org/badge/644048016.svg)](https://zenodo.org/badge/latestdoi/644048016)
 
+Data Repository
 
-## Versions and platforms
-*Fiji macro* was used with ImageJ 1.53t
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.8326749.svg)](https://doi.org/10.5281/zenodo.8326749)
 
-*Cellprofiler pipeline* was run on CellProfiler version 4.2.5
+# Computational Protocols
 
-*R* code was run with R version 4.3.0 (2023-04-21)
+For each step run the commands indicated in code blocks from the "chlamy-comparison" directory.
+
+**Protocol for segmenting cells and taking measurements of 2D morphology:**
+
+
+1. Download data from Zenodo. This will be a directory called "experiments" with subdirectories and image data.
+
+        curl -JLO https://zenodo.org/record/8326749/files/experiments.zip?download=1
+
+2. Crop "pools" of swimming cells from raw videos using the Fiji Macro batch_interactive_crop_savetif.v.0.0. Samples of raw videos are contained in the the "experiments/{experiment}/raw_sample" directories. Choose "experiments" as the input directory. [Link to Fiji macro](https://github.com/Arcadia-Science/chlamy-comparison-private/blob/main/code/FIJI/batch_interactive_crop_savetif.ijm)
```

Comment:
You can also use relative paths for this instead full GitHub links, it makes working locally (when not everything is on GitHub) much easier